* src/useradd.c: Fixed wrong format string.
* lib/gshadow.c: Removed declaration of unused variable.

===================================================================
--- src/useradd.c	(revision 3052)
+++ src/useradd.c	(revision 3054)
@@ -684,7 +684,7 @@
 static void usage (void)
 {
 	(void) fprintf (stderr,
-	                _("Usage: useradd [options] LOGIN\n"
+	                _("Usage: %s [options] LOGIN\n"
 	                  "\n"
 	                  "Options:\n"),
 	                Prog);

===================================================================
--- lib/gshadow.c	(revision 3052)
+++ lib/gshadow.c	(revision 3054)
@@ -216,7 +216,6 @@
 	static char *buf = NULL;
 
 	char *cp;
-	struct sgrp *ret;
 
 	if (0 == buflen) {
 		buf = (char *) malloc (BUFSIZ);
